PublicGitArchive/pga/README.md

```diff
@@ -16,20 +16,31 @@ In the meanwhile you'll need to compile this tool.
 
 There are three subcommands in `pga`: `list`, `get`, and `siva`.
 
+### Datasets
+
+Two datasets are exposed through this tool, and both the `list` and `get` command can be used to explore and retrieve them. To do so, you must specify with a keyword which dataset you want to work on :
+
+- `siva`: The original Public Git Archive dataset, made up of Siva files.
```

Comment:
```suggestion
- `siva`: The original Public Git Archive dataset, made of Siva files.
```
to make up = to imagine and tell stories :D

Reply:
done :p